skalvi/models.py

```diff
@@ -15,6 +15,7 @@ class UserProfile(models.Model):
     type = models.CharField(max_length=5)  # A = admin, P = parent, C = child, etc.
     phone = models.CharField(max_length=8, null=True, blank=True)  # 90 90 99 09 <-- gives length 8
     profile_name = models.CharField(max_length=30)
+    is_active = models.BooleanField(default=False)
```

Comment:
Can only one of the "user group" be signed in at a given time, our can multiple, say father and child be active simultaneously?